test/features/test_organized_edge_detection.cpp

```diff
@@ -0,0 +1,149 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2020-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#include <pcl/features/organized_edge_detection.h>
+#include <pcl/test/gtest.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+namespace {
+class OrganizedPlaneDetectionTestFixture : public ::testing::Test {
+protected:
+  const int INNER_SQUARE_EDGE_LENGTH = 50;
+  const int OUTER_SQUARE_EDGE_LENGTH = INNER_SQUARE_EDGE_LENGTH * 2;
+  const float SYNTHETIC_CLOUD_BASE_DEPTH = 2.0;
+  const float SYNTHETIC_CLOUD_DEPTH_DISCONTINUITY = .02f;
+  const float SYNTHETIC_CLOUD_RESOLUTION = 0.01f;
+
+  pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_;
+  std::set<int> outer_perimeter_;
+  std::set<int> inner_perimeter_;
```

Comment:
do a find replace for `std::set<int>` -> `std::set<index_t>`.  currently it's an alias for `std::set<int>` but can be changed during configuration.